### PR TITLE
v1.31.0 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where using `Share...` option to post highlighted text to the app threw an error.
+ - Fixed an issue where slash commands with long descriptions had their description text truncated in the slash command autocomplete.
+ - Fixed an issue where the username that created the webhook was shown on webhook posts instead of the name of the bot.
+ - Fixed an issue where posts on the same thread appeared to be from different threads since the "...commented on [Thread Title]" was shown on all posts.
+ - Fixed an issue where the system message for "Edit Channel Purpose" rendered markdown.
+ - Fixed an issue where "Someone" username appeared in prefetched channels.
+ - Fixed an issue where pre-fetching of unread channels included archived channels.
```
Comment (amyblais) [inline CHANGELOG.md:22]: These probably need a better description / more info to make it more clear for end users.

Comment (enahum) [reply CHANGELOG.md:23]: This one is not a fix, is actually a perceived perf improvement, but lets confirm first if it stays in or if we are going to revert it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Improvements
+ 
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where using `Share...` option to post highlighted text to the app threw an error.
```
Comment (enahum) [inline CHANGELOG.md:16]: iOS specific

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+ - Fixed an issue where slash commands with long descriptions had their description text truncated in the slash command autocomplete.
+ - Fixed an issue where users could not swipe up to dismiss in-app push notifications.
+ - Fixed an issue where the username that created the webhook was shown on webhook posts instead of the name of the bot.
+ - Fixed an issue where posts on the same thread appeared to be from different threads since the "...commented on [Thread Title]" was shown on all posts in the thread.
+ - Fixed an issue where the system message for "Edit Channel Purpose" rendered markdown.
+ - Fixed an issue where "Someone" username appeared in prefetched channels.
```
Comment (enahum) [inline CHANGELOG.md:22]: no need to add this to the changelog

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+ 
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where slash commands with long descriptions had their description text truncated in the slash command autocomplete.
+ - Fixed an issue where users could not swipe up to dismiss in-app push notifications.
+ - Fixed an issue where the username that created the webhook was shown on webhook posts instead of the name of the bot.
+ - Fixed an issue where posts on the same thread appeared to be from different threads since the "...commented on [Thread Title]" was shown on all posts in the thread.
+ - Fixed an issue where the system message for "Edit Channel Purpose" rendered markdown.
+ - Fixed an issue where pre-fetching of unread channels included archived channels.
```
Comment (esethna) [inline CHANGELOG.md:21]: This should be a highlight.

Improved network reliability and channel switching time for unread channels by fetching new posts as soon as the app reconnects.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.31.0 Release
+- Release Date: May 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
```
Comment (esethna) [inline CHANGELOG.md:8]: Should we add a note at the top of this list about recommending upgrading for servers not on v5.19, with a link to the blog post?

We should be on the lookout for opportunities to make this plug in ALL our documentation

Comment (amyblais) [reply CHANGELOG.md:8]: I can add a link to the blog afterwards when it has been published.

**Comment by amyblais** [comment]
@enahum Can you help review and merge this?

**Comment by enahum** [comment]
We need @esethna approval before merging

**Pull request merged**